app/components/course-card.tsx

```diff
@@ -1,8 +1,16 @@
 import * as React from 'react'
+import {motion, Variants} from 'framer-motion'
 import {ArrowIcon} from './icons/arrow-icon'
 import {H2} from './typography'
 import {ButtonLink} from './button'
 
+const MotionButtonLink = motion(ButtonLink)
```

Comment:
I've added the animation stuff in this component, instead of in `Button`, because this seems to be the only case of this type of button. 

If we'd made the abstraction in `Button`, we get something as complicated as `ArrowLink`.